modules/sequence/src/main/scala/lucuma/odb/sequence/gmos/longslit/Config.scala

```diff
@@ -206,6 +209,26 @@ object Config {
         explicitSpatialOffsets
       )
 
+    def reconcile(a: GmosNorth, modes: List[ObservingMode]): Option[GmosNorth] =
+      modes.headOption match {
+        case None                                                  =>
+          a.some
+
+        case Some(b@GmosNorth(_, _, _, _, _, _, _, _, _, _, _, _)) =>
+          if (a === b)
+            reconcile(a, modes.tail)
+          else {
+            val x  = a.xBin min b.xBin
+            val aʹ = a.copy(explicitXBin = none, defaultXBin = x)
+            val bʹ = b.copy(explicitXBin = none, defaultXBin = x)
+            if (aʹ === bʹ) reconcile(aʹ, modes.tail) else none
+          }
```

Comment:
It's a bit grim to repeat this for north and south, but I couldn't figure out a good way of making it work for both.